habana/1.11.0/ubi8-python-3.8/Pipfile

```diff
@@ -0,0 +1,64 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[dev-packages]
+
+[packages]
+# Habana requirements
+habana_media_loader = "==1.11.0.587"
+
+# TensorFlow and useful extensions
+habana-tensorflow = "==1.11.0.587"
+habana-horovod = "==1.11.0.587"
+tensorflow = "==2.12.1"
+tensorflow-io = "==0.32.0"
+tensorboard = "~=2.12.2"
+tf2onnx = "~= 1.13.0"
+
+# Pytorch and useful extension
+mpi4py = "==3.1.4"
+habana-pyhlml = "==1.11.0.587"
+
+# Below pkgs are required for pytorch-fork
+cffi = "==1.15.1"
+typing-extensions = ">=4.5.0"
+numpy = "==1.23.5"
+ninja = ">=1.11.1"
+pyyaml = "==6.0"
+pybind11 = "==2.10.4"
+mkl = "==2023.1"
+mkl-include = "==2023.1"
+
+# Parent image requirements to maintain cohesion
+elyra-pipeline-editor-extension = "~=3.15.0"
+elyra-python-editor-extension = "~=3.15.0"
+elyra-code-snippet-extension = "~=3.15.0"
+kfp-tekton = "<1.6.0"
+boto3 = "~=1.26.69"
+jupyter-bokeh = "~=3.0.5"
+kafka-python = "~=2.0.2"
+matplotlib = "~=3.6.3"
+pandas = "~=1.5.3"
+plotly = "~=5.13.0"
+scikit-learn = "~=1.2.1"
+scipy = "~=1.10.0"
+jupyterlab-lsp = "~=3.10.2"
+jupyterlab-widgets = "~=3.0.5"
+jupyter-resource-usage = "~=0.6.0"
+jupyterlab-s3-browser = "~=0.10.1"
```

Comment:
I think we have to consider deprecating this package, as it won't be included in the next release.